mmv1/products/privateca/api.yaml

```diff
@@ -845,11 +1182,358 @@ objects:
+               Otherwise, any key may be used.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'rsa'
+                  description: |
+                    Describes an RSA key that may be used in a Certificate issued from a CaPool.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'minModulusSize'
+                      description: |
+                         The minimum allowed RSA modulus size, in bits. If this is not set, or if set to zero, the
+                         service-level min RSA modulus size will continue to apply.
+                    - !ruby/object:Api::Type::String
+                      name: 'maxModulusSize'
+                      description: |
+                         The maximum allowed RSA modulus size, in bits. If this is not set, or if set to zero, the
+                         service will not enforce an explicit upper bound on RSA modulus sizes.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'ellipticCurve'
+                  description: |
+                    Represents an allowed Elliptic Curve key type.
+                  properties:
+                    - !ruby/object:Api::Type::Enum
+                      name: 'signatureAlgorithm'
+                      description: |
+                         The algorithm used.
+                      required: true
+                      values:
+                        - ECDSA_P256
+                        - ECDSA_P384
+                        - EDDSA_25519
+          - !ruby/object:Api::Type::String
+            name: 'maximumLifetime'
+            description: |
+              The maximum lifetime allowed for issued Certificates. Note that if the issuing CertificateAuthority
+              expires before a Certificate's requested maximumLifetime, the effective lifetime will be explicitly truncated to match it.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'allowedIssuanceModes'
+            description: |
+              IssuanceModes specifies the allowed ways in which Certificates may be requested from this CaPool.
+            properties:
+              - !ruby/object:Api::Type::Boolean
+                name: 'allowCsrBasedIssuance'
+                required: true
+                description: |
+                   When true, allows callers to create Certificates by specifying a CSR.
+              - !ruby/object:Api::Type::Boolean
+                name: 'allowConfigBasedIssuance'
+                required: true
+                description: |
+                   When true, allows callers to create Certificates by specifying a CertificateConfig.
+          - !ruby/object:Api::Type::NestedObject
```

Comment:
should there also be a nested object for passthroughExtensions?

Reply:
We decided to omit these from the initial implementation